src/amount.h

```diff
@@ -15,6 +15,8 @@ typedef int64_t CAmount;
 
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
+static const size_t KB = 1000;
+static const size_t PRECISION_MULTIPLIER = KB;
```

Comment:
This name is too general, people will confuse this with something innate to bitcoin (like COIN). `FEERATE_PRECISION_MULTIPLIER` maybe?